pulsar-broker/src/test/java/org/apache/pulsar/broker/loadbalance/LinuxInfoUtilsTest.java

```diff
@@ -0,0 +1,31 @@
+package org.apache.pulsar.broker.loadbalance;
+
+import static org.testng.Assert.assertEquals;
+import lombok.extern.slf4j.Slf4j;
+import org.testng.annotations.Test;
+
+@Slf4j
+@Test(groups = "broker")
+public class LinuxInfoUtilsTest {
+
+    /**
+     * simulate reading contents in /sys/fs/cgroup/cpuset/cpuset.cpus to get the number of Cpus.
+     */
+    @Test
+    public void testGetTotalCpuCount() {
+        int totalCpuCount=0;
+        String CPU_CPUSET_CPUS = "0-2,16,20-30";
+        String[] ranges = CPU_CPUSET_CPUS.split(",");
+        for (String range : ranges) {
```

Comment:
Can you try Mockito.mockStatic like the below to mock the file read part? I don't think we should copy the logic in the unit test.(when the logic changes in the code, this unit test cannot catch it)
 
```
MockedStatic<[LinuxInfoUtils> linuxInfoUtils = Mockito.mockStatic(LinuxInfoUtils.class))
            linuxInfoUtils.when(() -> LinuxInfoUtils.readTrimStringFromFile(any())
                    .thenReturn("0-2,16,20-30");
LinuxInfoUtilsTest.getTotalCpuCount();
```

Reply:
OK

Reply:
done.